### ISSUE TITLE
Problem login in as administrator

### ISSUE THREAD

**Issue by fmanh2** [grafana/grafana#3394]
I am getting deseratehere. I have installed a vanilla docker image with grafana. The setup is pain vanilla with grafanas admin account set to admin:admin (user/password). I have double and triple checked this and the log says it is indeed using admin/admin for user and password (never mind the security at this point in time).

This grafana i running in a docker environment on an ubuntu machine. It start the logs looks nice and I am able to signup a user and login with that user on the system. Sweet. But it refuses, no matter what I do to allow me to login as administrator which basically renders the installation totally useless.

I do not see any obvious problems here. It should be noted that the whole setup is running on an headerless server, which means that I access everything from a web browser on another machine (same network). Is the admin user restricted to localhost or anything like that?

**Comment by torkelo** [comment]
admin/admin should work. It does when I spin up a new docker instance.

**Comment by fmanh2** [comment]
I did some further invetigation iun to the matter, using the docker grafana/grafana locally (on my lap top) and pointing the browser to 192.168.99.100:3000 everything runs perfectly. Pointing to the redirected port 3000 I will only be able to login with 'normal' users.I am not suggesting that there is anything wrong in the image (I am currently more leaning to me doing something fishy). I was thinking on whether or not admin users here not allowed on non local ports or something along those lines. But that also seems odd... Any suggestions is most welcome. Otherwise this is a great product! I should point out that I have tried it under ubuntu 15.10, and locally on a windows machine. Docker is kind of a bummer on windows machine (barely working), so that could be affecting the situation as well.
